PrincessBrideTrivia/PrincessBrideTrivia/Program.cs

```diff
@@ -1,92 +1,133 @@
-﻿namespace PrincessBrideTrivia;
-
-public class Program
+﻿namespace PrincessBrideTrivia
 {
-    public static void Main(string[] args)
+    public class Program
     {
-        string filePath = GetFilePath();
-        Question[] questions = LoadQuestions(filePath);
-
-        int numberCorrect = 0;
-        for (int i = 0; i < questions.Length; i++)
+        public static void Main(string[] args)
         {
-            bool result = AskQuestion(questions[i]);
-            if (result)
+            string filePath = GetFilePath();
+            Question[] questions = LoadQuestions(filePath);
+
+            List<Question> incorrectQuestions = new List<Question>();
+
+            int numberCorrect = 0;
+            // Ask each question once
+            for (int i = 0; i < questions.Length; i++)
             {
-                numberCorrect++;
+                bool result = AskQuestion(questions[i]);
+                if (result)
+                {
+                    numberCorrect++;
+                }
+                else
+                {
+                    // Store the questions answered wrong
+                    incorrectQuestions.Add(questions[i]);
+                }
             }
-        }
-        Console.WriteLine("You got " + GetPercentCorrect(numberCorrect, questions.Length) + " correct");
-    }
 
-    public static string GetPercentCorrect(int numberCorrectAnswers, int numberOfQuestions)
-    {
-        return (numberCorrectAnswers / numberOfQuestions * 100) + "%";
-    }
+            Console.WriteLine("You got " + GetPercentCorrect(numberCorrect, questions.Length) + " correct.");
 
-    public static bool AskQuestion(Question question)
-    {
-        DisplayQuestion(question);
+            // While loop to keep asking the incorrect questions till they get them all right
+            while (incorrectQuestions.Count > 0)
+            {
+                Console.WriteLine("\nLets make sure you know all the correct answers. You have " + incorrectQuestions.Count + " questions to try again!\n");
```

Comment:
nit: Favor `Enviroment.NewLine` over `\n`

```suggestion
                Console.WriteLine($"{Enviroment.NewLine}Lets make sure you know all the correct answers. You have " + incorrectQuestions.Count + " questions to try again!{Enviroment.NewLine}");
```